guides/doc-Release_Notes/topics/katello-4.14.2.adoc

```diff
@@ -0,0 +1,22 @@
+= Katello 4.14.2
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1914[Redmine]
+
+== Katello
+
+=== API
+
+* pass:[apidoc doesn't allow to unset CV/LCE of an activationkey] - https://projects.theforeman.org/issues/38008[#38008]
+
+=== Content Views
```

Comment:
```suggestion
=== Content views
```

Reply:
I swear I thought you were on PTO because of your status in Slack 🤣 Updated